bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/obscomp/SPInstObsComp.java

```diff
@@ -443,9 +443,9 @@ public Instrument getInstrument() {
 
 
         m.put(Instrument.Flamingos2, Duration.ofSeconds(10));
+        m.put(Instrument.Ghost,      Duration.ofSeconds(10));
```

Comment:
This is odd, why does order matter?

Reply:
Totally agree, it makes no sense

Reply:
Hmm, this fixes some test failure?  What was it?

Reply:
It is odd but the previous last commit made the itc tests fail

Reply:
I bet it is unrelated.  It's good to have it sorted like this, but I would have to think if you put it back the tests would still run?

Reply:
I bet it is unrelated.  It's good to have it sorted like this, but I would have to think if you put it back the tests would still run?